pkg/sentry/fs/proc/sys_net_state.go

```diff
@@ -40,3 +43,15 @@ func (s *tcpSack) afterLoad() {
 		}
 	}
 }
+
+// beforeSave is invoked by stateify.
+func (ipf *ipForwarding) beforeSave() {
+	ipf.enabled = ipf.stack.Forwarding(ipv4.ProtocolNumber)
+}
+
+// afterLoad is invoked by stateify.
+func (ipf *ipForwarding) afterLoad() {
+	if err := ipf.stack.SetForwarding(ipv4.ProtocolNumber, ipf.enabled); err != nil {
+		panic(fmt.Sprintf("failed to set previous IPv4 forwarding configuration [%v]: %v", ipf.enabled, err))
```

Comment:
I'd probably just log the error rather than panicking. do we do this anywhere else in afterLoad?

Reply:
Ok, if that's the right thing to do here.